framework/config/config.py

```diff
@@ -84,6 +72,13 @@ def initialize_attributes(self):
         for type in CONFIG_TYPES:
             self.Config[type] = {}
 
+    def SelectConfigFilePath(self, file_path):
```

Comment:
Make the function names to use the snakecase - `select_config_path`. The other function names in the file will be change in a short while.